decidim-surveys/lib/decidim/surveys/component.rb

```diff
@@ -36,6 +36,11 @@
     raise "Can't destroy this component when there are survey answers" if survey_answers_for_component.any?
   end
 
+  component.register_resource(:survey) do |resource|
+    resource.model_class_name = "Decidim::Surveys::Survey"
+    resource.searchable = true
```

Comment:
This seems to break the global search.

Try the global search with a matching search term with one of the surveys.

Can you also add this into specs to catch this won't happen in the future?